orbit/pkg/packaging/windows_templates.go

```diff
@@ -115,26 +111,25 @@ var windowsWixTemplate = template.Must(template.New("").Option("missingkey=error
                   Return="check"
                   Impersonate="no" />
 
-   <SetProperty Id="CA_StopOrbit"
-                 Before ="CA_StopOrbit"
+   <SetProperty Id="CA_RemoveOrbit"
+                 Before ="CA_RemoveOrbit"
                  Sequence="execute"
-                 Value='&quot;[POWERSHELLEXE]&quot; -NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass -File "[ORBITROOT]installer_utils.ps1" -stopOrbit' />
+                 Value='&quot;[POWERSHELLEXE]&quot; -NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass -File "[ORBITROOT]installer_utils.ps1" -uninstallOrbit' />
```

Comment:
So the change is making it so that it always uses this approach to delete `C:\Program Files\Orbit`? (instead of the `C_ORBITROOT_REMOVAL` above)

Reply:
Yes, exactly. The change is around using the `-uninstallOrbit` powershell custom action (handled [here](https://github.com/fleetdm/fleet/blob/dd519fd303d83c1ec8a3984d0214bd40acbd68e6/orbit/pkg/packaging/windows_templates.go#L596)), instead of C_ORBITROOT_REMOVAL ([RemoveFolderEx()](https://wixtoolset.org/docs/v3/xsd/util/removefolderex/)) as we noticed few issues with this WIX helper (runtime generated files not being deleted for example)

Reply:
OK, just tested it and it removes the whole Orbit directory as expected 🎉

Reply:
Awesome!